### PR TITLE
Opensearch

### PR DESCRIPTION
**Pull request by wgresshoff**
:heart: Thank you for your contribution!

### Description
This PR removes support of Elastixsearch versions <7 and adds suport of Opensearch 1

### Checklist
:rotating_light: Please review the [guidelines for contributing](https://invenio.readthedocs.io/en/latest/community/contributing/contribution-guide.html) to Invenio.

- [ ] Did you create different commits for logically separate things? Check [the format](https://invenio.readthedocs.io/en/latest/community/contributing/contribution-guide.html#commit-messages) of the commit(s) message(s).
- [ ] [InvenioRDM specific] Did you follow the style guide?  Check [the documentation](https://inveniordm.docs.cern.ch/maintenance/styleguide/).
- [ ] [InvenioRDM specific] Did you add translations (i18n) for new labels, backend and frontend? Check [the documentation](https://inveniordm.docs.cern.ch/develop/topics/i18n/).
- [ ] [InvenioRDM specific] Did you add accessibility (a11y) tags? Check [the documentation](https://inveniordm.docs.cern.ch/maintenance/accessibility/).
- [ ] Did you add documentation? We all love well-documented frameworks, right?
- [ ] Did you include third-party code (a library, package, etc.)? If yes:
    - [ ] Make sure that its open source license is compatible with Invenio (*note:* GPL and AGPL licensed code is not accepted)
    - [ ] Did you respect the license conditions (normally you need to keep the copyright and license header)?

> Green light on all GitHub Actions status checks is required in order to merge your PR. Check your code additions will fail neither code linting checks nor tests and will not decrease test coverage.

#### Copyright

- [x] Identify who is the copyright holder (a legal entity - i.e. person or organization - normally this is your employer) for the produced code.
- [x] Make sure that you have updated the copyright headers for touched files (non-trivial contributions >15 lines).

See [copyright and license policy](https://invenio.readthedocs.io/en/latest/community/contributing/maintainers-guide/copyright-policy.html) for details.

#### Reminder

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

If you ticked all the boxes above, you're good to go! :rocket:

### CODE REVIEW

File: docker_services_cli/__init__.py [hunk 1]
```diff
@@ -11,21 +13,21 @@
...
-    $ export ES_VERSION=ES_7_LATEST
+    $ export OPENSEARCH_VERSION=OS_1_LATEST
 
 Then it simply needs to boot up the services. Note that if no version was
 exported in the environment, the CLI will use the default values set in
 ``env.py``.
 
 .. code-block:: console
 
-    $ docker-services-cli up es postgresql redis
```
Comment (max-moser) [inline docker_services_cli/__init__.py:28]: this part of the documentation didn't work (anymore?)

File: docker_services_cli/docker-services.yml [hunk 2]
```diff
@@ -47,6 +47,25 @@ services:
...
       - "9200:9200"
       - "9300:9300"
+  opensearch:
+    image: opensearchproject/opensearch:${OPENSEARCH_VERSION}
+    restart: "unless-stopped"
+    environment:
+      - bootstrap.memory_lock=true
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
+      - plugins.security.disabled=true
```
Comment (max-moser) [inline docker_services_cli/docker-services.yml:57]: OS 1.x enables SSL and auth (`admin:admin`) per default... which i think is also the case with ES 8.x

**Review by ntarocco — APPROVED** [review]

**Comment by ntarocco** [comment]
@ppanero can you have a quick look to this one? IMO, it can be merged and released :)

Thanks @max-moser! 🚀

File: docker_services_cli/__init__.py [hunk 3]
```diff
@@ -11,21 +13,21 @@
...
+    $ docker-services-cli up --cache os --db postgresql --cache redis
```
Comment (ppanero) [inline docker_services_cli/__init__.py:30]:
```suggestion
    $ docker-services-cli up --search os --db postgresql --cache redis
```

File: docker_services_cli/config.py [hunk 4]
```diff
@@ -25,20 +25,28 @@
 versions before loading a given service's version.
 """
 
-DOCKER_SERVICES_FILEPATH = "docker_services_cli/docker-services.yml"
+DOCKER_SERVICES_FILEPATH = "docker-services.yml"
```
Comment (ppanero) [inline docker_services_cli/config.py:28]: hmm how was it working before? or why is this needed? I'm fearful it won't work via CLI on test?

**Review by ppanero — APPROVED** [review]
LGTM except for the file path, Im wondering if there was something always broken and if not how is it finding the file now 🤔

File: docker_services_cli/cli.py [hunk 5]
```diff
@@ -125,7 +131,7 @@ def __init__(self, filepath, verbose):
     "--filepath",
     "-f",
     required=False,
-    default=f"{get_python_lib()}/docker_services_cli/docker-services.yml",
+    default=f"{_get_module_path()}/docker-services.yml",
```
Comment (max-moser) [inline docker_services_cli/cli.py:134]: @ppanero this here is related to your question regarding the path...
previously, the path to the `docker-services.yml` would be constructed via `distutils.sysconfig.get_python_lib()`, which will return the location where the Python source files are installed "normally":
```python
In [2]: get_python_lib()
Out[2]: '/home/mmoser/.local/share/virtualenvs/tu-data-3eTT44w9/lib/python3.8/site-packages'
```

However, if you install `docker-services-cli` in editable mode (`pip install -e /path/to/docker-services-cli`, then this would be obviously wrong:
The `yml` file would then be located in `/path/to/docker-services-cli/docker_services_cli/docker-services.yml`, but the old way of constructing the path would still give back `/home/mmoser/.local/share/virtualenvs/tu-data-3eTT44w9/lib/python3.8/site-packages/docker_services_cli/docker-services.yml`.

TL;DR: The old way of looking the `yml` file worked for normal installs (`pip install docker-services-cli`), but not for editable installs (`pip install -e ...`).

File: docker_services_cli/config.py (hunk 4 above)
Comment (max-moser) [reply docker_services_cli/config.py:28]: see the other [comment](https://github.com/inveniosoftware/docker-services-cli/pull/26/files#r926655763)

File: docker_services_cli/cli.py (hunk 5 above)
Comment (ppanero) [reply docker_services_cli/cli.py:134]: Thanks for taking the time to explain this. PR LGTM!
